README.md

```diff
@@ -10,34 +10,40 @@
   <b>
     <a href="https://rudderstack.com">Website</a>
     ·
-    <a href="">Documentation</a>
+    <a href="https://rudderstack.com/docs/features/transformations/libraries/#rudderstack-libraries">Documentation</a>
     ·
     <a href="https://rudderstack.com/join-rudderstack-slack-community">Community Slack</a>
   </b>
 </p>
 
 ---
 
-# \*\*Repo Name\*\*
+# RudderStack libraries
 
-\*\*Repo description\*\*
+RudderStack provides some libraries out of the box which you can directly import into your transformations. This repo contains the relevant code and implementation-specific details for these libraries.
 
-## Overview
+Currently, RudderStack provides the following libraries:
 
-\*\*Describe what the software does.\*\*
+- [Encrypt/decrypt PII](#encryptdecrypt-pii)
+- [User agent parser](#user-agent-parser)
+- [Hashing PII](#hashing-pii)
 
-## Features
+## Encrypt/decrypt PII
 
-\*\*Describe the key features, if necessary.\*\*
+This library lets you encrypt and decrypt PII including those that are stored in cookies. Refer to the library's code [here](https://github.com/rudderlabs/rudder-libraries/tree/main/libraries/encrypt/v1).
 
-## Getting started
+## User agent parser
 
-\*\*Describe how to use the software.\*\*
+The [Parse user agent transformation template](https://rudderstack.com/docs/features/transformations/templates/#parse-user-agent) imports this library to add the user's browser, engine, OS, device, and CPU-related information to the events. Refer to the library's code [here](https://github.com/rudderlabs/rudder-libraries/tree/main/libraries/userAgentParser/v1).
+
```

Comment:
paste code here rather than navigating to docs

Reply:
Makes sense - done.